src/Umbraco.Core/Persistence/Dtos/NodeDto.cs

```diff
@@ -20,7 +20,7 @@ internal class NodeDto
 
         [Column("uniqueId")]
         [NullSetting(NullSetting = NullSettings.NotNull)]
-        [Index(IndexTypes.UniqueNonClustered, Name = "IX_" + TableName + "_UniqueId")]
+        [Index(IndexTypes.UniqueNonClustered, Name = "IX_" + TableName + "_UniqueId", IncludeColumns = "parentId,level,path,sortOrder,trashed,nodeUser,text,createDate")]
```

Comment:
This index can't be modified using the current code as it's a unique constraint used for a foreign key. It may be better to add a second index.

Reply:
Breaks upgrade from 8.6.4

Reply:
The migration is fixed in the latest rev